lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -77,6 +77,26 @@ def verify(credit_card, options = {})
         end
       end
 
+      def store(payment, options = {})
+        MultiResponse.run do |r|
+          if options[:customer_id].nil?
+            customer_id = r.process { commit('customers', create_customer_request(payment, options)) }.params['id']
+            break r unless r.success?
+          else
+            customer_id = options[:customer_id]
+          end
+          instrument_identifier = r.process { create_instrument_identifier_request(payment, options) }
+          break instrument_identifier unless instrument_identifier.success?
+
+          r.process { create_customer_payment_instrument_request(payment, options, customer_id, instrument_identifier) }
+        end
+      end
+
+      def unstore(third_party_token)
```

Comment:
```suggestion
      def unstore(stored_token)
```
TPT is specific terminology